src/Workspaces/Core/Portable/Shared/Extensions/SyntaxTriviaListExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Shared.Extensions
+{
+    internal static class SyntaxTriviaListExtensions
+    {
+       public static SyntaxTrivia LastOrDefault(this SyntaxTriviaList triviaList)
+            => triviaList.Count == 0 ? default(SyntaxTrivia) : triviaList.Last();
```

Comment:
:bulb: `default`

:bulb: Maybe move the `Any` extension from `Microsoft.CodeAnalysis.CSharp.Extensions.SyntaxTriviaListExtensions` to here?